.changelogs/issue_2237.yml

```diff
@@ -0,0 +1,7 @@
+significance: minor
+type: dev
+comment: Added a helper function `llms_get_floats_rounding_precision()` for returning precision with filter `lifterlms_reporting_average_precision` for rounding off floating values in the reporting section for quizzes average column.
+links:
+  - "#2237"
+entry: Added helper function `llms_get_floats_rounding_precision()` to return precision for rounding off floating
```

Comment:
Can you add a reference to the newly added filter hook too? Thanks! <3
Like:
Added filter hook `lifterlms_floats_rounding_precision`, to filter bla bla bla ?

Reply:
Done! 👍